docs/source/en/tutorials/fast_diffusion.md

```diff
@@ -34,15 +28,23 @@ Then upgrade the other required libraries too:
 pip install -U transformers accelerate peft
 ```
 
-To benefit from the fastest kernels, use PyTorch nightly. You can find the installation instructions [here](https://pytorch.org/). 
+Install [PyTorch nightly](https://pytorch.org/) to benefit from the latest and fastest kernels:
+
+```bash
+pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu
+```
+
+<Tip>
+
+The results reported below are from a 80GB 400W A100 with its clock rate set to the maximum. <br>
 
-To report the numbers shown below, we used an 80GB 400W A100 with its clock rate set to the maximum.
+If you're interested in the full benchmarking code, take a look at [huggingface/diffusion-fast](https://github.com/huggingface/diffusion-fast).
 
-_This tutorial doesn't present the benchmarking code and focuses on how to perform the optimizations, instead. For the full benchmarking code, refer to: [https://github.com/huggingface/diffusion-fast](https://github.com/huggingface/diffusion-fast)._
+</Tip>
 
 ## Baseline
 
-Let's start with a baseline. Disable the use of a reduced precision and [`scaled_dot_product_attention`](../optimization/torch2.0):
+Let's start with a baseline. Disable reduced precision and [scaled dot product attention (SDPA)](../optimization/torch2.0#scaled-dot-product-attention):
```

Comment:
`scaled_dot_product_attention()` _is_ the name of the function and I think it makes sense to have it within backquotes.